### DISCUSSION TITLE
Not recieved education benifits it's been around 5 days after i got approved as student.

### DISCUSSION THREAD

**Discussion by Honeybhandari** [community/community#132346] [GitHub Education]
### Hi everyone,

What is the process furthur as i haven't recieved the benifits as students kindly guide my through the process.Thank you

**Comment by ghost** [comment]
Hi, it usually takes a few days, but sometimes it can take a few weeks, wait a few more days. When benefits become available, the status icon will turn purple.

**Reply by ibnunaufal** [reply]
did we get some notification to email or we should check the status manually if education benefits given?

**Reply by ghost** [reply]
I can't say for sure because I don't use it, but it will tell you that benefits are now available.
